src/bindings/python/src/pyopenvino/graph/partial_shape.cpp

```diff
@@ -145,17 +145,34 @@ void regclass_graph_PartialShape(py::module m) {
         )");
 
     shape.def(
-        "__eq__",
```

Comment:
why have you deleted this method?
They are correct -- `==` is defined for PartialShape [here](https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/partial_shape.hpp#L186).